### PR TITLE
K8SPXC-252 Added updating monitor users

### PR DESCRIPTION
**Pull request by nonemax**
[K8SPXC-252](https://jira.percona.com/browse/K8SPXC-252)

### CODE REVIEW

File: pkg/pxc/users/users.go [hunk 1]
```diff
@@ -55,6 +55,24 @@ func (u *Manager) UpdateUsersPass(users []SysUser) error {
 				}
 				return errors.Wrap(err, "update password")
 			}
+			if user.Name == "monitor" {
+				_, err = tx.Exec("GRANT SELECT, PROCESS, SUPER, REPLICATION CLIENT, RELOAD, CREATE USER ON *.* TO 'monitor'@?", host)
```
Comment (delgod) [inline pkg/pxc/users/users.go:59]: why CREATE USER is needed?

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |

File: pkg/pxc/users/users.go (hunk 1 above)
Comment (nonemax) [reply pkg/pxc/users/users.go]: removed https://github.com/percona/percona-xtradb-cluster-operator/pull/450/commits/abad5844a306f8953bacde563dbec1608caf9603

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | failed |
| big-data | failed |
| init-deploy | failed |
| storage | passed |
| scaling | failed |
| restore-to-encrypted-cluster | failed |
| limits | passed |
| scaling-proxysql | passed |
| demand-backup | passed |
| monitoring | passed |
| upgrade | failed |
| self-healing | failed |
| self-healing-advanced | passed |
| upgrade-consistency | passed |
| monitoring-2-0 | failed |
| security-context | passed |
| operator-self-healing | passed |
| affinity | passed |
| scheduled-backup | failed |
| demand-backup-encrypted-with-tls | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| recreate | passed |
| storage | passed |
| init-deploy | failed |
| scaling | passed |
| big-data | passed |
| limits | passed |
| monitoring | passed |
| self-healing | passed |
| scaling-proxysql | passed |
| restore-to-encrypted-cluster | passed |
| monitoring-2-0 | passed |
| upgrade | failed |
| self-healing-advanced | failed |
| demand-backup | passed |
| affinity | passed |
| operator-self-healing | failed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| scheduled-backup | passed |
| demand-backup-encrypted-with-tls | passed |
| upgrade-consistency | failed |

File: pkg/pxc/users/users.go [hunk 2]
```diff
@@ -99,6 +99,24 @@ func (u *Manager) UpdateUsersPass(users []SysUser) error {
 				}
 				return errors.Wrap(err, "update password")
 			}
+			if user.Name == "monitor" {
```
Comment (dAdAbird) [inline pkg/pxc/users/users.go:102]:
```go
func (u *Manager) UpdateUsersPass(users []SysUser) error {
...
```
This method not only updates passwords anymore

**Comment by delgod** [comment]
superseded by https://github.com/percona/percona-xtradb-cluster-operator/pull/452

**Pull request closed without merging**